Step-by-step-Tutorials/My-First-Orleans-Application.md

```diff
@@ -53,7 +53,9 @@ static void Main(string[] args)
         AppDomainInitializerArguments = args,
     });
 
-    Orleans.GrainClient.Initialize("DevTestClientConfiguration.xml");
+    // Orleans comes with a rich XML configuration but we're just going to setup a basic config
+    var config = new Orleans.Runtime.Configuration.ClientConfiguration();
+    GrainClient.Initialize(config);
```

Comment:
That is good to know! My main point of reference was the example in [here](dotnet.github.io/orleans/Step-by-step-Tutorials/Minimal-Orleans-Application#host--programcs). Then, we should change that example, too. Do you agree?